src/framework/preferences/js/Enactors.js

```diff
@@ -427,4 +427,244 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
+                var backgroundColor = $(element).css("background-color"); // the color can be is rgb or rgba format
+                if (backgroundColor.substr(0,4) === "rgba") {
+                    backgroundColor = backgroundColor.slice(5,-1);
+                }
+                else {
+                    backgroundColor = backgroundColor.slice(4,-1);
+                }
+                backgroundColor = backgroundColor.split(", ");
+                initialColorsDictionary.push({
+                    key: this,
+                    color: color,
+                    backgroundColor: backgroundColor
+                });
+            }
+            noMatchFlag = true;
+        });
+
+        var initialColors = $.extend(true, [], initialColorsDictionary);  // make a deep copy of the original array in order to prevent changes in the initial array
+        elements.each(function () {
+            // Finds the element in the dictionary
+            for (var count in initialColors) {
+                if (initialColors[count].key == this) {
+                    index = count;
+                    break;
+                }
+            }
+            var color = initialColors[index].color;  //get the color of the current component
+            color[2] = Math.round(color[2] * (1 - times)) + "";  //blue component
+            var colorToSet = "rgb(" + color[0] + ", " + color[1] + ", " + color[2] + ")";  //make the color in a "rbg(..,..,..)" format
+            $(this).css("color", colorToSet);
+            var backgroundColor = initialColors[index].backgroundColor;
+            backgroundColor[2] = Math.round(backgroundColor[2] * (1 - times)) + "";
+            if (backgroundColor[3] === undefined) {
+                var backgroundColorToSet = "rgb(" + backgroundColor[0] + ", " + backgroundColor[1] + ", " + backgroundColor[2] + ")";
+            }
+            else {
+                var backgroundColorToSet = "rgba(" + backgroundColor[0] + ", " + backgroundColor[1] + ", " + backgroundColor[2] + ", " + backgroundColor[3] + ")";
+            }
+            $(this).css("background-color", backgroundColorToSet);
+        });
+    };
+
+    fluid.prefs.enactor.blueColorFilter.addMutationObserver = function (applyFilter) {
+        // Uses document ready, because the script is added before the body
+        $(document).ready(function (){
+            var isSafari = /^((?!chrome|android).)*safari/i.test(navigator.userAgent);
+            if (isSafari) {
+                var observer = new WebKitMutationObserver(function(mutations) {
+                    mutations.forEach(function (mutation) {
+                        if (mutation.type === "childList") {
+                            // Changes in HTMLHtmlElement can be just hovering over elements, which should not trigger applyFilter
+                            if (changedDOMFlag === false && mutation.target != "[object HTMLHtmlElement]") {
```

Comment:
Use ```!==``` instead of ```!=```. Run ```grunt lint``` on this branch to get a linting report and fix them all. Thanks.